subjects/DPxAI/play-with-variables/README.md

```diff
@@ -0,0 +1,93 @@
+## Play with variables
+
+> Mindful AI mode
+
+### Context
+
+If things get a little hectic at times, take the time to get closer to your peers so that you can think, share and move forward together.
+
+> Keep Going!
+
+### AI-Powered Learning Techniques
+
+**Clarification Technique:**
+
+This type of prompt encourages the AI to explain a concept in detail, helping you gain a deeper understanding.
+
+> Find the examples across the subject ;)
+
+## Concepts
+
+### Escape characters
+
+**Quote delimiters** can be one of the tricky things to deal with.
+
+Since they are used for delimiting text, they need a trick to include them in
+our text.
+
+For example, we want a `'` _(single quote)_ in our text, but use them as
+delimiters:
+
+```js
+console.log('I keep trying , I can't give up! ')
+// too bad a single quote, ruined the quote, get it ? XD
```

Comment:
I would remove the `XD` at the end of the line

Reply:
Addressed